server/src/main/java/io/pravega/schemaregistry/server/rest/auth/AuthHandlerManager.java

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
Curious, can we not use the `AuthHandlerManager` from Pravega here? It'd be nice to avoid the duplication so that any change we end up making in there also applies here.

Reply:
The problem is, it assumes the `ServiceConfig` implements Controller's ServiceConfig.